packages/calypso-stripe/package.json

```diff
@@ -36,7 +36,7 @@
 	"homepage": "https://github.com/Automattic/wp-calypso/tree/HEAD/packages/calypso-stripe#readme",
 	"dependencies": {
 		"@stripe/react-stripe-js": "^1.4.1",
-		"@stripe/stripe-js": "^1.17.1",
+		"@stripe/stripe-js": "1.17.1",
```

Comment:
This needed to be pinned, because the semver range makes it compatible with Stripe 1.46. Unfortunately, our typescript types are _not_ compatible with Stripe 1.46. By pinning it, we make Calypso resolve Stripe 1.17 as it should.